### PR TITLE
Fix: image pull backoff

### PR DESCRIPTION
**Pull request by shinofara**
An error occurs because the image specified in manifest does not exist. Also, since latest does not exist, I fixed the tag.

### CODE REVIEW

File: deploy/manifests/deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: originissuer-control
       containers:
         # pushing to Docker Hub will be available after open sourcing
```
Comment (terinjokes) [inline deploy/manifests/deployment.yaml]: remove this line as well

**Review by terinjokes — COMMENTED** [review]
Thanks for your contribution, I was actually working on this change (along with a few other additions) now, since I now have the ability to push to Docker Hub.

I've left one comment here, but you'll probably run into a few more issues that came about from pulling this out of internal setup that I'm working on in e2e now.

**Comment by shinofara** [comment]
So that's it. I use this change, is that okay?

**Comment by terinjokes** [comment]
If you remove the link I commented on, I can merge this PR.

**Comment by terinjokes** [comment]
24702999240964d9a72cb736d4bac408e8f1f4f3

**Pull request closed without merging**